src/modules/cameras/camera-container.js

```diff
@@ -227,6 +227,9 @@ export default function(RANDO) {
 
     RANDO.CameraContainer.prototype.enableCamera = function (id) {
         var cameraIdElement = document.querySelector(".camera--" + id);
+        if (cameraIdElement === null) {
```

Comment:
replace by !cameraIdElement

Reply:
Thanks for the review 👍 

Why not, but what is the benefit?
According to the documentation, it seems correct to be explicit ⬇️ 
> The [Document](https://developer.mozilla.org/en-US/docs/Web/API/Document) method querySelector() returns the first [Element](https://developer.mozilla.org/en-US/docs/Web/API/Element) within the document that matches the specified selector, or group of selectors. If no matches are found, null is returned.